.github/workflows/build.yml

```
@@ -30,5 +30,5 @@ jobs:
           restore-keys: ${{ runner.os }}-maven-
                
       - name: Build with Maven
-        run: mvn test
+        run: mvn --no-transfer-progress --batch-mode clean test
```

```suggestion
        run: mvn --no-transfer-progress --batch-mode verify
```

I believe that's the right way here

---

Done